npm ERR! Could not resolve dependency: npm ERR! peer react@"^16.14.0" from react-flickity-component@3.6.1

#### [max@workstation Web]$ npm install
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: shahro@0.1.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.14.0" from react-flickity-component@3.6.1
npm ERR! node_modules/react-flickity-component
npm ERR!   react-flickity-component@"^3.6.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/max/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/max/.npm/_logs/2021-05-21T10_06_34_630Z-debug.log
```

Hi again;

No any solution and plan to solve or help me how can solve this?

Thanks in advance.

```
[max@workstation Projects]$ cd Web/
[max@workstation Web]$ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: shahro@0.1.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.14.0" from react-flickity-component@3.6.1
npm ERR! node_modules/react-flickity-component
npm ERR!   react-flickity-component@"^3.6.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/max/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/max/.npm/_logs/2021-05-28T12_08_01_991Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -42,8 +42,8 @@
   },
   "peerDependencies": {
     "flickity": "^2.2.1",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0"
+    "react": ">= 16.14.0 < 18",
+    "react-dom": ">= 16.14.0 < 18"
   },
   "jest": {
     "testURL": "http://localhost/"

